lcm-java/lcm-spy.1

```diff
@@ -1,52 +1,37 @@
-.TH lcm-spy 1 2009-07-28 "LCM" "Lightweight Communications and Marshalling (LCM)"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.3.
+.TH LCM-SPY "1" "November 2024" "lcm-spy 1.5.0" "Lightweight Communications and Marshalling (LCM)"
 .SH NAME
-lcm-spy - traffic inspection tool
+lcm-spy \- LCM spy: traffic inspection tool
 .SH SYNOPSIS
 .TP 5
 \fBlcm-spy \fI[options]\fR
-
 .SH DESCRIPTION
+usage: lcm\-spy [options]
 .PP
-\fBlcm-spy\fR is the Lightweight Communications and Marshalling traffic 
-inspection utility.  It is a graphical tool for viewing messages received on 
-an LCM network, and is analogous to tools like Ethereal/Wireshark and tcpdump
+lcm\-spy is the Lightweight Communications and Marshalling traffic
+inspection utility.  It is a graphical tool for viewing messages received on
+an LCM network, and is analagous to tools like Ethereal/Wireshark and tcpdump
 in that it is able to inspect all LCM messages received and provide information
 and statistics on the channels used.
-
-When given appropriate LCM type definitions, \fBlcm-spy\fR is able to
+.PP
+When given appropriate LCM type definitions, lcm\-spy is able to
 automatically detect and decode messages, and can display the individual fields
-of recognized messages.  \fBlcm-spy\fR is limited to displaying statistics for
+of recognized messages.  lcm\-spy is limited to displaying statistics for
 unrecognized messages.
-
-\fBlcm-spy\fR collects the following statistics for each channel:
-.IP - 2
-Channel name
-.IP - 2
-Number of messages received
-.IP - 2
-Message Rate (Hz)
-.IP - 2
-Bandwidth (KB/s)
-
-On its own, this information can often be used to verify that messages are
-being transmitted on the expected channels, and at the expected data rates.
```

Comment:
I tried moving it to the help message but the bullet points format poorly in the man page if I do that. I could move this to `lcm-java/lcm-spy.1.h2m` to preserve it. The only potential downside is it will come first in that section now.

edf1ba32e65cb0e0bc7888a83ec7cc9aaf87d074

Reply:
Ah, I see.  Sorry, I think I prefer the previous behavior in this case.